### ISSUE TITLE
After using yabai, all tiled windows seem to be on top by default, causing small pop-ups like Chrome bookmarks, WeChat emojis, and similar windows to be covered. How can I solve this issue?

### ISSUE THREAD

**Issue by torch132** [koekeishiya/yabai#1798]
- Macbook pro M1
- version：v13.2.1

**Comment by carlosvigil** [comment]
Same issue since the last update in May. I uninstalled, not worth the trouble anymore.

**Comment by sum-rock** [comment]
Also having this issue after a recent Yabai upgrade (not sure which version jump specifically). I notice this most on Chrome with bookmark folders in the bookmark toolbar but also observe the behavior with slack notifications. Slack notifications now (almost) always appear behind windows. Hovering or clicking a book mark folder will not display the dropdown menu. Clicking on the settings panel in the top right corner of Chrome does not open the drop down menu either. Interestingly, if you restart Chrome the issue goes away until you visit a few pages and do some scrolling. Then the issue returns.

I have also tried to disable window boarders, transparency, and boarder blur. Nothing seems to make a difference. You can see my yabairc [here](https://github.com/sum-rock/just-sum-nix/blob/master/homes/modules/yabai/yabairc).

I absolutely love Yabai and cannot exist without it so this is very very annoying. I want to make sure I'm sending the maintainers some love an appreciation though. If anyone has a thread to pull on, I would love to help get it resolved. For now, I just wanted to chime in here to make sure we knew that its an issue for more people.

**Comment by jakucermak** [comment]
Happened to me as well with yabai version of yabai-v5.0.6 on Mac Mac14,10 OS: macOS 13.4.1 22F82 arm64. I resolved the issue with stopping yabai service and restart my Mac. For comparison my .yabairc:
```
#!/usr/bin/env sh

#
# for this to work you must configure sudo such that
# it will be able to run the command without password
#
# see this wiki page for information:
#  - https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition
#
 yabai -m signal --add event=dock_did_restart action="sudo yabai --load-sa"
 sudo yabai --load-sa
#

# global settings
yabai -m config                                 \
    mouse_follows_focus          off            \
    focus_follows_mouse          off            \
    window_origin_display        default        \
    window_placement             second_child   \
    window_zoom_persist          on             \
    window_topmost               on            \
    window_shadow                on             \
    window_animation_duration    0.0            \
    window_animation_frame_rate  120            \
    window_opacity_duration      0.0            \
    active_window_opacity        1.0            \
    normal_window_opacity        0.90           \
    window_opacity               off            \
    insert_feedback_color        0xffd75f5f     \
    active_window_border_color   0xff775759     \
    normal_window_border_color   0xff555555     \
    window_border_width          4              \
    window_border_radius         12             \
    window_border_blur           off            \
    window_border_hidpi          on             \
    window_border                off            \
    split_ratio                  0.50           \
    split_type                   auto           \
    auto_balance                 off            \
    top_padding                  13             \
    bottom_padding               13             \
    left_padding                 13             \
    right_padding                13             \
    window_gap                   10             \
    layout                       bsp            \
    mouse_modifier               fn             \
    mouse_action1                move           \
    mouse_action2                resize         \
    mouse_drop_action            swap


yabai -m rule --add app="^Google Chrome|Firefox|Safari|Arc$" space=1
yabai -m rule --add app="^Neovide$" space=2
yabai -m rule --add app="^iTerm2|Warp$" space=3
yabai -m rule --add app="^Music$" space=4
yabai -m rule --add app="^Mail|Fantastical$" space=6


yabai -m signal --add event=window_focused action="sketchybar --trigger window_focus"
yabai -m signal --add event=window_created action="sketchybar --trigger windows_on_spaces"
yabai -m signal --add event=window_destroyed action="sketchybar --trigger windows_on_spaces"

yabai -m rule --add app="^System Settings$" manage=off

echo "yabai configuration loaded.."
```

**Comment by gldtn** [comment]
Try disabling topmost on your config file.

```
# floating windows are always on top (default: off)
yabai -m config window_topmost off
```

It currently does not work well with some programs like Chrome, if your having problem just disable it. For some people this seems to work, but I could not get it to work right.

Hope it helps!

**Comment by sum-rock** [comment]
@gldtn You're my personal hero right now. Thanks to you're comment I am no longer considering turning my MacBook into a $2K frisbee. This worked for me, I hope it works for others!

**Comment by koekeishiya** [comment]
Fixed in v6.0.0, #1887
